libs/portals/admin/document-provider/src/screens/InstitutionSingleDocumentProvider/InstitutionSingleDocumentProvider.tsx

```diff
@@ -0,0 +1,232 @@
+import { useEffect, useState } from 'react'
+import { useLocale } from '@island.is/localization'
+
+import {
+  Box,
+  Breadcrumbs,
+  DatePicker,
+  GridColumn,
+  GridContainer,
+  GridRow,
+  Pagination,
+} from '@island.is/island-ui/core'
+import { m } from '../../lib/messages'
+import { InstitutionDocumentProviderDashboard } from './InstitutionDocumentProviderDashboard'
+import { IntroHeader } from '@island.is/portals/core'
+import { useGetStatisticsOverviewByProviderId } from '../../shared/useGetStatisticsOverviewByProviderId'
+import { useParams } from 'react-router-dom'
+import { useGetProviderStatisticsBreakdownByProviderId } from '../../shared/useGetProviderStatisticsBreakdownByProviderId'
+import { DocumentProviderStatisticsTable } from '../../components/DocumentProviderStatisticsTable/DocumentProviderStatisticsTable'
+import { useGetProviderStatisticsBreakdownWCategoriesByProviderId } from '../../shared/useGetProviderStatisticsBreakdownWCategoriesByProviderId'
+import { useGetProvidersByNationalId } from '../../shared/useGetProvidersByNationalId'
+import { useUserInfo } from '@island.is/react-spa/bff'
+import { DocumentProvidersNavigation } from '../../components/DocumentProvidersNavigation/DocumentProvidersNavigation'
+import { DocumentProvidersLoading } from '../../components/DocumentProvidersLoading/DocumentProvidersLoading'
+import {
+  StatisticBoxList,
+} from '../../components/StatisticBoxList/StatisticBoxList'
+import { CategoryStatisticsSortBy } from '@island.is/api/schema'
+import { DocumentProviderPaths } from '../../lib/paths'
+import { StatisticsBoxData } from '../../lib/types'
+
+const SingleDocumentProvider = () => {
+  const today = new Date()
+  // Set toDate to the first day of the current month
+  const firstOfThisMonth = new Date(today.getFullYear(), today.getMonth(), 1)
+  // Set fromDate to the first day of the same month, one year ago
+  const firstOfLastYearMonth = new Date(
+    today.getFullYear() - 1,
+    today.getMonth(),
+    1,
+  )
+  const [fromDate, setFromDate] = useState<Date | undefined>(
+    firstOfLastYearMonth,
+  )
+  const [toDate, setToDate] = useState<Date | undefined>(firstOfThisMonth)
+  const [numPages, setNumPages] = useState(0)
+  const [pageNumber, setPageNumber] = useState(1)
+  const [pageSize] = useState(10)
+  const { formatMessage } = useLocale()
+
+  const { providerId } = useParams<{ providerId: string }>()
+
+  const user = useUserInfo()
+
+  const { loading, statistics } = useGetStatisticsOverviewByProviderId(
+    user.profile.nationalId,
```

Comment:
Could the queries be combined in a bigger one maybe? With some field resolvers to reduce the amount of duplication going on